packages/attestation-proxy/src/index.ts

```diff
@@ -33,3 +31,39 @@ export const proxyReveal = functions.https.onCall(async (data, context) => {
     throw new functions.https.HttpsError('unknown', error.message)
   }
 })
+
+export const proxyRevealStatus = functions.https.onCall(async (data, context) => {
+  // refuse requests that do not have basic auth
+  checkAuth(context)
+
+  const { attestationServiceUrl, phoneNumber, account, issuer, pepper } = data
+  try {
+    const urlParams = new URLSearchParams({
+      phoneNumber,
+      salt: pepper ?? '',
```

Comment:
Out of scope here but just a nit, would be nice to have salt renamed in attestation service too: https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/io.ts#L109

Since there's been changes there anyway seems like there's an opportunity to do that (in a backwards compat way) before the next release. cc @aslawson @timmoreton

Reply:
we've already published the attestation service, so unfortunately can't do it this iteration :/